core/shared/src/main/scala/scalaz/zio/stm/STM.scala

```diff
@@ -194,6 +194,11 @@ final class STM[+E, +A] private[stm] (
         }
     )
 
+  /**
+    * Ignores any failure and return `Unit`
+    */
+  final def ignore: STM[Nothing, Unit] = self.either.unit
```

Comment:
Looks great! Be sure to place it alphabetically in the list of methods. 👍

Reply:
Done